doc/sphinx/advanced_methods.rst

```diff
@@ -1105,6 +1103,318 @@ with respect to the underlying object, the triangle 123 needs to be
+   moderate charge densities. At higher valencies or densities,
+   overcharging and layering effects can occur, which lead to
+   non-monotonic charge densities and potentials, that can not be
+   covered by a mean-field model such as Poisson-Boltzmann or this one.
+
+   Even in salt free systems containing only counter ions, the
+   counter-ion densities close to highly charged objects can be
+   overestimated when neglecting excluded volume effects. Decades of the
+   application of Poisson-Boltzmann theory to systems of electrolytic
+   solutions, however, show that those conditions are fulfilled for
+   monovalent salt ions (such as sodium chloride or potassium chloride)
+   at experimentally realizable concentrations.
+
+-  Electrodynamic and magnetic effects play no role. Electrolytic
+   solutions fulfill those conditions as long as they don’t contain
+   magnetic particles.
+
+-  The diffusion coefficient is a scalar, which means there can not be
+   any cross-diffusion. Additionally, the diffusive behavior has been
+   deduced using a formalism relying on the notion of a local
+   equilibrium. The resulting diffusion equation, however, is known to
+   be valid also far from equilibrium.
+
+-  The temperature is constant throughout the system.
+
+-  The density fluxes instantaneously relax to their local equilibrium
+   values. Obviously one can not extract information about processes on
+   length and time scales not covered by the model, such as dielectric
+   spectra at frequencies, high enough that they correspond to times
+   faster than the diffusive time scales of the charged species.
+
+.. _Setup:
+
+Setup
+~~~~~
+
+.. _Initialization:
+
+Initialization
+^^^^^^^^^^^^^^
+::
+
+    import espressomd
+    sys = espressomd.System()
+    sys.box_l = [10, 20, 30]
+    sys.time_step = 0.0
+    sys.cell_system.skin = 0.4
+    ek = espressomd.electrokinetics.Electrokinetics(agrid = 1.0, lb_density = 1.0, 
+    viscosity = 1.0, friction = 1.0, T =1.0, prefactor = 1.0, stencil = 'linkcentered', advection = True, fluid_coupling = 'friction')
+    sys.actors.add(ek)
+
+.. note:: `Feature ELECTROKINETICS and LB_GPU required`
+
+The above is a minimal example how to initialize the LB fluid, and
+it is very similar to the Lattice-Boltzmann command in set-up. We
+therefore refer the reader to Chapter :ref:`Lattice-Boltzmann` for details on the
+implementation of LB in and describe only the major differences here.
+
+The first major difference with the LB implementation is that the
+electrokinetics set-up is a Graphics Processing Unit (GPU) only
+implementation. There is no Central Processing Unit (CPU) version, and
+at this time there are no plans to make a CPU version available in the
+future. To use the electrokinetics features it is therefore imperative
+that your computer contains a CUDA capable GPU which is sufficiently
+modern.
+
+To set up a proper LB fluid using this command one has to specify at
```

Comment:
|es| missing

Reply:
Where is the |es| missing?

Reply:
i misread the sentence, nothing is missing